src/tests/cwrapper/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+project(test_cwrapper)
+
+add_executable(${PROJECT_NAME} test_cwrapper.c)
```

Comment:
I verified that cmake calls C for this file:
```
Building C object src/tests/cwrapper/CMakeFiles/test_cwrapper.dir/test_cwrapper.c.o
Linking CXX executable test_cwrapper
```
and C++ for linking it, which is needed due to the C++ standard library and other stuff. So all is good.